README.md

```diff
@@ -96,18 +94,67 @@ Resources have defined attributes.  Attributes are explicitly defined.  An error
 
 A subset of these `attributes` may be a `reference`, which is simply a special type of attribute that references another resource.  You can retrieve the references of a resource by calling `LedgerSync::Customer.references`.
 
+#### Custom Attributes
+
+Some ledgers (e.g. NetSuite) allow for custom attributes (or "fields"), which can vary by account and user.  To allow for custom attributes, you can create new resources, ledger serializers (see below), and validation contracts (see below).  Assuming your ledger supports the string attribute `foo` for customers, you could do the following:
+
+```ruby
+class CustomCustomer < LedgerSync::Customer
+  attribute :foo, type: LedgerSync::Type::String
+end
+```
+
 ### Adaptors
 
 Adaptors are ledger-specific ruby objects that contain all the logic to authenticate to a ledger, perform ledger-specific operations, and validate resources based on the requirements of the ledger.  Adaptors contain a number of useful objects:
 
 - adaptor
+- serializers
 - operations
 - searchers
 
 #### Adaptor
 
 The adaptor handles authentication and requests to the ledger.  Each adaptors initializer will vary based on the needs of that ledger.
 
+#### Serializers
+
+Operations depend on `LedgerSync::Adaptor::LedgerSerializer`s to serialize and deserialize objects.  Most resources have a default serializer per adaptor which may acts as both a serializer and deserializer.  You can provide custom serializers, which is necessary when working with custom attributes.  For example, given the following:
+- `custom_resource` that is a `LedgerSync::Customer` (see above)
+- the attribute `foo` is used in both the request and response bodies
+- using the `NetSuite` adaptor
+
+you could implement custom serializers using the following code:
+
+```ruby
+# Extend the existing one or use
+# LedgerSync::Adaptors::NetSuite::LedgerSerializer or, more generically,
```

Comment:
Should `LedgerSync::Adaptors::NetSuite::LedgerSerializer` be `LedgerSync::Adaptors::NetSuite::Customer::LedgerSerializer` ?

Reply:
Not necessarily.  The inheritance looks like this:

` LedgerSync::Adaptors::NetSuite::Customer::LedgerSerializer <  LedgerSync::Adaptors::NetSuite::LedgerSerializer <  LedgerSync::Adaptors::LedgerSerializer`

It really just depends on what you're looking to accomplish.

I updated the readme on this to hopefully be a bit more clear.